docs/operations/export_api.md

```diff
@@ -0,0 +1,422 @@
+# Export API
+
```

Comment:
It would be nice here to have a bit of intro text so the reader can quickly scan from search results to see if this is what they're after.

Also, small nit, you go from H1 straight down to H3, bypassing H2... You could promote most of your headers by one or add some big idea chapter headings like Prerequisites.

Reply:
Reason I go from H1 to H3 is that H2 renders almost the same as H1 and it is hard to tell the difference between them. I can definitely add some intro text to explain the point of this.